css/src/components/tabs.scss

```diff
@@ -0,0 +1,115 @@
+$tabs-padding: $font-size-9 $font-size-5 !default;
```

Comment:
inner component padding should use em so that the outer component can set font size and have easily scalable sizes

Reply:
updated